examples/03-Maxwell/Maxwell_Transformer_Coreloss.py

```diff
@@ -0,0 +1,136 @@
+"""
+Maxwell 3D: Transformer
+-----------------------
+This example shows how you can use PyAEDT to set core loss given a set
+of BP curves at different frequencies.
+"""
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
+
+from pyaedt import downloads
+from pyaedt import generate_unique_folder_name
+from pyaedt import Maxwell3d
+from pyaedt.generic.constants import unit_converter
+from pyaedt.generic.general_methods import read_csv
+
+
+def _convert_tuple_to_float(curves_csv):
```

Comment:
@MaxJPRey @Samuelopez-ansys @Alberto-DM  any advice here for this provate method needed to remove the header of the csv file and convert all string numbers to float? I'm not sure this is the correct place to put this but it's not even general enough to put it in general_methods.py.

Reply:
sorted!